docs/esp8266/tutorial/intro.rst

```diff
@@ -99,16 +99,16 @@ And then deploy the new firmware using::
 
     esptool.py --port /dev/ttyUSB0 --baud 460800 write_flash --flash_size=detect 0 esp8266-20170108-v1.8.7.bin
 
-You might need to change the "port" setting to something else relevant for your
-PC.  You may also need to reduce the baudrate if you get errors when flashing
-(eg down to 115200).  The filename of the firmware should also match the file
-that you have.
+You must use the `port <https://github.com/espressif/esptool#serial-port>`_
+setting for your operating system. You may also need to reduce the baudrate if
+you get errors when flashing (eg down to 115200).  The filename of the firmware
+should also match the file that you have.
 
 For some boards with a particular FlashROM configuration (e.g. some variants of
 a NodeMCU board) you may need to use the following command to deploy
 the firmware (note the ``-fm dio`` option)::
 
-    esptool.py --port /dev/ttyUSB0 --baud 460800 write_flash --flash_size=detect -fm dio 0 esp8266-20170108-v1.8.7.bin
+    esptool.py --port COM1 --baud 460800 write_flash --flash_size=detect -fm dio 0 esp8266-20170108-v1.8.7.bin
```

Comment:
Unless this is mostly used on Windows there doesn't seem to be a compelling reason to change this?

Reply:
It's trying to give Windows a hint about what a `port` might look like

Reply:
I understand, but by doing so the hint for unix users is gone and that one is arguably more interesting because unlike COMx it's harder to learn by heart. Could add a line of text saying Windows users can for example use `--port COM1 ` ?

Reply:
No it's not gone, it's still in the original example 5 lines above. Anyway, commit reformatting info as Notes below.